File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/Help UserControl/FullSreenControlBar.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports C1.Win.TouchToolKit

Namespace C1ZoomPages.Help_Forms
    Partial Public Class ControlBarUserControl03
        Inherits UserControl
        Public Sub New()
            InitializeComponent()
        End Sub

        Private zoomObj As C1Zoom
        Protected Overrides Sub OnLoad(e As EventArgs)
            MyBase.OnLoad(e)

            zoomObj = Me.gcZoomCommandProvider1.OwnerGcZoom

            button5_Click(Nothing, EventArgs.Empty)
		End Sub

		Private Sub button5_Click(sender As Object, e As EventArgs)
			If zoomObj.FullScreenMode = FullScreenMode.FullScreen OrElse zoomObj.FullScreenMode = FullScreenMode.FullScreenWithTaskBar Then
				button5.Text = "Normal form"
			Else
				button5.Text = "Full screen"
			End If
		End Sub

		Private Sub button4_Click(sender As Object, e As EventArgs)
			TryCast(zoomObj.Target, Form).Close()
		End Sub
	End Class
End Namespace

File: NetFramework/TouchToolkit/CS/TouchToolkitDemo/GcZoomPages/Help UserControl/FullSreenControlBar.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using C1.Win.TouchToolKit;

namespace TouchToolkitDemo.C1ZoomPages.Help_Forms
{
    public partial class ControlBarUserControl03 : UserControl
    {
        public ControlBarUserControl03()
        {
            InitializeComponent();
        }

        C1Zoom zoomObj;
        protected override void OnLoad(EventArgs e)
        {
            base.OnLoad(e);

            zoomObj = this.C1ZoomCommandProvider1.OwnerGcZoom;

            button5_Click(null, EventArgs.Empty);
        }

        private void button5_Click(object sender, EventArgs e)
        {
            if (zoomObj.FullScreenMode == FullScreenMode.FullScreen || zoomObj.FullScreenMode == FullScreenMode.FullScreenWithTaskBar)
            {
                button5.Text = "Normal form";
            }
            else
            {
                button5.Text = "Full screen";
            }
        }

        private void button4_Click(object sender, EventArgs e)
        {
            (zoomObj.Target as Form).Close();
        }
    }
}
